### PR TITLE
docs: quickstart fuel wallet integration

### LINKED ISSUE #85: docs: quickstart fuel wallet integration
Current documentation uses the CLI wallet and hardcodes the wallet's secret.
Since [Fuel Wallet](https://wallet.fuel.network/) is already out _(as an alpha)_ should be included at least in the quickstart.

### PR DESCRIPTION
**Pull request by cold-briu**
resolves #85 

This approach uses the "useFuel" and "useIsConencted" hooks from the [fuel-wallet/sdk documentation](https://wallet.fuel.network/docs/react/) and adds a third custom hook named "useWallet" the last one being added by me. 

The implementation (and the quickstart) can be cleaner by optimizing the hooks and adding them directly into the sdk. 
However that's out of this PR's scope.

### CODE REVIEW

File: beta2-quickstart-master/frontend/src/hooks/useWallet.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { WalletLocked } from 'fuels';
+import React, { useEffect, useMemo, useState } from 'react'
+import { useFuel } from './useFuel';
+import { useIsConnected } from './useIsConnected';
+
+const useWallet = () => {
```
Comment (luizstacio) [inline beta2-quickstart-master/frontend/src/hooks/useWallet.tsx]:
```suggestion
export const useWallet = () => {
```

File: beta2-quickstart-master/frontend/src/hooks/useWallet.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	const [account, setAccount] = useState<string>();
+	const [wallet, setWallet] = useState<WalletLocked | null>(null);
+	const isConnected = useIsConnected();
+	const [Fuel] = useFuel();
+
+	useEffect(() => {
+		async function getAccounts() {
+			const currentAccount = await Fuel.currentAccount();
```
Comment (luizstacio) [inline beta2-quickstart-master/frontend/src/hooks/useWallet.tsx]: The current account can present issues if the user doesn't allow the existing account to connect. I suggest using `const [account] = await fuel.accounts()`.

**Review by luizstacio — CHANGES_REQUESTED** [review]
As this is, a quickstart application is good to reduce the number of files the user needs to create. It would be great to maybe create this as an advanced example, maybe something under.

Examples -> React with Hooks.

**Comment by cold-briu** [comment]
> As this is, a quickstart application is good to reduce the number of files the user needs to create. It would be great to maybe create this as an advanced example, maybe something under.

Hello there! 
Thanks for the feedback. 

I think the same way as you, however App component gets really complex without the hooks.
It might be better to add the hooks inside the npm package.

**Pull request closed without merging**